doc/conf.py

```diff
@@ -302,3 +302,14 @@
     }
 
 numpydoc_class_members_toctree = False
+
+def touch_example_backreferences(app, what, name, obj, options, lines):
+    # generate empty examples files, so that we don't get
+    # inclusion errors if there are no examples for a class / module
+    examples_path = os.path.join(app.srcdir, "generated", "%s.examples" % name)
+    if not os.path.exists(examples_path):
+        # touch file
+        open(examples_path, 'w').close()
```

Comment:
Why did you need to do this? If you are running latest `sphinx-gallery` it should not be necessary as they have lines like this in their code.